os/mm/kmm_heap/kmm_calloc.c

```diff
@@ -146,15 +147,14 @@ void *kmm_calloc_at(int heap_index, size_t n, size_t elem_size)
 
 FAR void *kmm_calloc(size_t n, size_t elem_size)
 {
+	size_t caller_retaddr;
 #ifdef CONFIG_DEBUG_MM_HEAPINFO
-	ARCH_GET_RET_ADDRESS
-#else
-	size_t retaddr = 0;
+	ARCH_GET_RET_ADDRESS(caller_retaddr)
 #endif
 	if (n == 0 || elem_size == 0) {
 		return NULL;
 	}
-	return kheap_calloc(n, elem_size, retaddr);
+	return kheap_calloc(n, elem_size, caller_retaddr);
```

Comment:
When CONFIG_DEBUG_MM_HEAPINFO is disabled, I think SVACE says warning for un-initialized variable. How do you think?

Reply:
I will add initialization of 0 for all.